04-intro-to-js/README.md

```diff
@@ -1,14 +1,14 @@
 ## Making our Webpages Interactive! 
 
-So as we just observed, not having javascript makes webpages **essentially unusable** (in most cases!)
+So as we just observed, not having JavaScript makes webpages **essentially unusable** (in most cases!)
   
-At this point, you're probably asking "Well, how do I actually harness the power of Javascript to make interactive websites and apps?", and for that, you've come to the right place!  
+At this point, you're probably asking "Well, how do I actually harness the power of JavaScript to make interactive websites and apps?", and for that, you've come to the right place!  
 
-There are two main ways to incorporate javascript into your HTML file, and they both involve the \<script> tag.  
+There are two main ways to incorporate JavaScript into your HTML file, and they both involve the \<script> tag.  
 (The \<script> tag is simply a way to let the browser know that whatever is inside should be **interpreted as JavaScript code**) 
 
 1. You can link an external .js file by specifying the *src* (source) *attribute*  
-```<script src="script.js"> </>```  
+``` <script src="script.js"> </>```  
```

Comment:
Hey Chase, awesome work on the React Native course. Quick note on your formatting:

If you want to format a code snippet **in the same line as the rest of your text, or in a paragraph**, then surround it with backticks like \`this\` (renders as `this`).

If you want to format a code snippet **in a separate block**, then surround it with three backticks on a newline:

\`\`\`
\<Like this\>
\`\`\`

Reply:
Thanks Leo!